Moodis/Moodis/Feature/Login/Register/RegisterForm.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Moodis.Feature.Login.Register
+{
+    public partial class RegisterForm : Form
+    {
+        LoginViewModel loginViewModel = new LoginViewModel();
```

Comment:
Maybe it would be better to create a separate ViewModel for RegisterForm? Now it is quite confusing, that LoginViewModel is responsible not just for login, but also for registration